modules/virt-creating-configmap.adoc

```diff
@@ -0,0 +1,264 @@
+    openshift.io/support-url: https://github.com/kubevirt/common-templates/issues
+    tags: hidden,kubevirt,virtualmachine,windows
+    template.kubevirt.io/editable: |
+      /objects[0].spec.template.spec.domain.cpu.cores
+      /objects[0].spec.template.spec.domain.resources.requests.memory
+      /objects[0].spec.template.spec.domain.devices.disks
+      /objects[0].spec.template.spec.volumes
+      /objects[0].spec.template.spec.networks
+    template.kubevirt.io/version: v1alpha1
+    template.openshift.io/bindable: "false"
+    validations: |
+      [
+        {
+          "name": "minimal-required-memory",
+          "path": "jsonpath::.spec.domain.resources.requests.memory",
+          "rule": "integer",
+          "message": "This VM requires more memory.",
+          "min": 536870912
+        }
+      ]
+  labels:
+    flavor.template.kubevirt.io/medium: "true"
+    os.template.kubevirt.io/windows: "true"
+    template.kubevirt.io/type: base
+    template.kubevirt.io/version: v0.11.2
+    workload.template.kubevirt.io/server: "true"
+  name: custom-windows-server-medium-v0.7.0
+  namespace: openshift
+objects:
+- apiVersion: kubevirt.io/v1alpha3
+  kind: VirtualMachine
+  metadata:
+    labels:
+      app: ${NAME}
+      vm.kubevirt.io/template: custom-windows-server-medium-v0.7.0
+      vm.kubevirt.io/template.revision: "1"
+      vm.kubevirt.io/template.version: v0.11.2
+    name: ${NAME}
+  spec:
+    running: false
+    template:
+      metadata:
+        labels:
+          kubevirt.io/domain: ${NAME}
+          kubevirt.io/size: medium
+      spec:
+        domain:
+          clock:
+            timer:
+              hpet:
+                present: false
+              hyperv: {}
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+            utc: {}
+          cpu:
+            cores: 1
+            sockets: 1
+            threads: 1
+          devices:
+            disks:
+            - disk:
+                bus: sata
+              name: rootdisk
+            interfaces:
+            - masquerade: {}
+              model: e1000e
+              name: default
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              spinlocks:
+                spinlocks: 8191
+              vapic: {}
+          machine:
+            type: pc-q35-rhel8.2.0
+          resources:
+            requests:
+              memory: 4Gi
+        evictionStrategy: LiveMigrate
+        networks:
+        - name: default
+          pod: {}
+        terminationGracePeriodSeconds: 0
+        volumes:
+        - name: rootdisk
+          persistentVolumeClaim:
+            claimName: ${PVCNAME}
+parameters:
+- description: VM name
+  from: windows-[a-z0-9]{6}
+  generate: expression
+  name: NAME
+- description: Name of the PVC with the disk image
+  name: PVCNAME
+  required: true
+EOF
+----
+
+. Create a custom ConfigMap:
```

Comment:
I don't think we should mix custom configMap examples with the windows work around or CNV-2.4.

Can you please put the Windows work around after this section, 
and in the configMap only mention the specific windows mapping?

And I would leave the bellow example containing the existing mapping, as detailed here:
https://github.com/kubevirt/vm-import-operator/blob/master/pkg/os/os-mapper-provider.go#L101

Reply:
I don't want to change the structure of the section too much for a temporary workaround. I would prefer to have the Windows workaround as part of the main procedure for 2.4 and then remove the windows workaround and add the regular mapping for 2.5. 

Alternatively, I could add a separate section for the Windows workaround and add a callout to the existing mapping so that the user doesn't use it by mistake for Windows. That way, I just tag that section for 2.4. I might do that anyway because it would ensure that the Windows workaround only appears for 2.4 and doesn't appear in any other version. I need to think about this.

Reply:
I just noticed something else. That whole custom configmap section is only for 2.4, so it makes no difference whether I put in the existing mapping or not! After 2.4, no one is going to create a custom configmap, not for Windows or other OS. :-)